package.json

```diff
@@ -38,26 +22,13 @@
     ]
   },
   "scripts": {
-    "build": "yarn clean && yarn build:scss && yarn build:css && yarn lint:copyright --fix lib/css/* && yarn build:html && yarn copy-files",
-    "build:scss": "cpx \"./src/**/*.scss\" ./lib/scss",
-    "build:css": "node ./scripts/generateCss.js src lib/css",
-    "build:html": "node ./scripts/generateHtml.js backstop/tests backstop/minified",
-    "build:watch": "yarn build && concurrently -n scss,html -k \"yarn build:watch:scss\" \"yarn build:watch:html\"",
-    "build:watch:scss": "chokidar \"src/**/*\" -c \"yarn build:scss && yarn build:css\"",
-    "build:watch:html": "chokidar \"backstop/tests/**/*\" -c \"yarn build:html\"",
-    "copy-files": "cpx package.json lib && cpx README.md lib",
-    "createComponent": "node ./scripts/createComponent.js",
-    "clean": "rimraf ./lib ./backstop/results/bitmaps_test",
-    "lint": "stylelint \"src/**/*.scss\"",
-    "lint:copyright": "node scripts/copyrightLinter.js",
-    "approve": "backstop --config=./backstop/backstop.js approve",
-    "test": "yarn build && yarn test:ci",
-    "test:ci": "backstop --config=./backstop/backstop.js test --docker",
-    "clean:images": "node scripts/removeOldTestImages.js",
-    "print": "node scripts/print.js",
-    "audit:ci": "audit-ci --moderate --report-type=full",
-    "build-docker": "node scripts/buildDocker.js"
+    "build": "turbo run build",
+    "dev": "turbo run dev",
+    "test": "turbo run test",
+    "lint": "turbo run lint",
+    "lint:copyright": "node packages/shared/copyrightLinter.js"
   },
+  "dependencies": {},
   "devDependencies": {
```

Comment:
Shouldn't a big part of these go to iTwinUI-CSS package.json?

Reply:
Moved the CSS-specific ones into itwinui-css